charts/README.md

```diff
@@ -7,124 +7,18 @@
 - Prerequisites
   - Kubernetes version is between 1.14 and 1.20.
   - The version of Prometheus Operator is v0.49.0.
-- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [minikube](https://minikube.sigs.k8s.io/docs/start/).
+- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [Minikube](https://minikube.sigs.k8s.io/docs/start/).
 - Requirements：
-  - Worker >= 4 vCPU 8 GB of Memory.
+  - For TKE: Worker >= 4 vCPU 8 GB of Memory.
```

Comment:
pr #15  reduces the component request esource. Do we specify at least how many resources are recommended for the production environment and how much is required for the test environment?